src/test/java/com/exasol/errorreporting/PlaceholderMatcherTest.java

```diff
@@ -0,0 +1,51 @@
+package com.exasol.errorreporting;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import org.junit.jupiter.api.Test;
+
+class PlaceholderMatcherTest {
+    @Test
+    void testMatching() {
+        final Iterable<Placeholder> matcher = PlaceholderMatcher.findPlaceholders("{{a}} {{b}}");
+        final List<String> placeholders = StreamSupport.stream(matcher.spliterator(), false).map(Placeholder::getName)
+                .collect(Collectors.toList());
+        assertThat(placeholders, contains("a", "b"));
+    }
+
+    @Test
+    void testCallHasNextMoreOften() {
```

Comment:
I think this test is not relevant anymore

Reply:
yea but I would like to have it in case someone (maybe future me) breaks it when refactoring